### PR TITLE
Backend.NetStandard: remove project

### PR DESCRIPTION
**Pull request by aarani**
The non legacy version of GWallet backend is now NETStandard 2.0 so we no longer need the extra GWallet.Backend.NetStandard project that we previously needed to build XF frontends.

### CODE REVIEW

File: gwallet-legacy.sln [hunk 1]
```diff
@@ -2,7 +2,7 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27130.0
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "GWallet.Backend", "src\GWallet.Backend\GWallet.Backend-legacy.fsproj", "{96F9B3E5-11F8-4F5F-AADC-51D0D995B3D2}"
+Project("{F2A71F9B-5D33-465A-A702-920D77279786}") = "GWallet.Backend.Legacy", "src\GWallet.Backend\GWallet.Backend-legacy.fsproj", "{96F9B3E5-11F8-4F5F-AADC-51D0D995B3D2}"
```
Comment (knocte) [inline gwallet-legacy.sln:5]: @aarani I only have 1 single issue with this PR and it's this ^ , what is GWallet.Backend.Legacy? I dont think that's the name of the assembly or the project (please fix this not only here, there are 2 occurrences or more)

Comment (aarani) [reply gwallet-legacy.sln:5]: No, this is the project name in solution, you can't have the same name for two projects in the same solution (VS breaks), so one had to be changed. This name doesn't really do anything as far as I know the Assembly name is specified in the fsproj file.

Comment (aarani) [reply gwallet-legacy.sln:5]: > No, this is the project name in solution, you can't have the same name for two projects in the same solution (VS breaks), so one had to be changed. This name doesn't really do anything as far as I know the Assembly name is specified in the fsproj file.

@knocte

Comment (knocte) [reply gwallet-legacy.sln:5]: @aarani ok please include this info in the commit msg (and re-run the broken CI)

File: src/GWallet.Backend/GWallet.Backend.fsproj [hunk 2]
```diff
@@ -58,6 +58,7 @@
     <Folder Include="Ether\" />
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="4.7.0" />
```
Comment (knocte) [inline src/GWallet.Backend/GWallet.Backend.fsproj:61]: @aarani what is this? if it's not a mistake it probably needs to be explained in commit msg too

**Pull request merged**